FEM/fem_ele_vec.cpp

```diff
@@ -2435,7 +2453,162 @@ void CFiniteElementVec::GlobalAssembly_RHS()
 				De->multi(strain_ne, dstress);
 				for (i = 0; i < ns; i++) // JT: This was commented. It shouldn't be.
 					dstrain[i] += strain_ne[i];
+		}
+			if (smat->Creep_mode == 1001) //BURGERS.
+			{
+				//get total strains at time t and current iteration i
+				double *strain_t, *strain_curr;
+				strain_t = new double[6]; strain_curr = new double[6];
```

Comment:
Yes, this is quite inefficient. This is testing a mapping scheme. This step won't be necessary in ogs6 and is confined here to these two models. I would ask to leave this for the moment.